db/db.go

```diff
@@ -6,18 +6,20 @@ import (
 	"google-scraper/models"
 
 	"github.com/beego/beego/v2/client/orm"
-	"github.com/beego/beego/v2/core/logs"
 	"github.com/beego/beego/v2/server/web"
 	_ "github.com/lib/pq"
 )
 
 func init() {
-	dbUrl, _ := web.AppConfig.String("dbUrl")
+	dbUrl, err := web.AppConfig.String("dbUrl")
+	if err != nil {
+		fmt.Println("Postgres database source is not found: ", err)
```

Comment:
IMO, This error handling (and `orm.RegisterDriver` as well) should not just only print because app cannot running without starting the database.
Maybe we can use [`log.Fatal()`](https://golang.org/pkg/log/#Fatal) instead.

Reply:
Beego uses own log package, so I used it `logs.Critical`